From: Tim Gardner <tim.gardner@canonical.com>
Date: Mon, 31 Mar 2008 14:52:26 +0000 (-0600)
Subject: UBUNTU: SAUCE: Blacklist Bluetooth Dell Wireless 370 for SCO MTU
X-Git-Url: http://kernel.ubuntu.com/git?p=ubuntu%2Fubuntu-intrepid.git;a=commitdiff_plain;h=ebac64c14591dd425b8442a3380b4df67ebad572

UBUNTU: SAUCE: Blacklist Bluetooth Dell Wireless 370 for SCO MTU
OriginalAuthor: Mario Limonciello <Mario_Limonciello@Dell.com>
Bug: #209715

Add Dell Wireless 370 device to blacklist to enable SCO MTU workaround.

Signed-off-by: Mario Limonciello <Mario_Limonciello@Dell.com>
Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
---

Index: 2008-devel/drivers/bluetooth/hci_usb.c
===================================================================
--- 2008-devel.orig/drivers/bluetooth/hci_usb.c
+++ 2008-devel/drivers/bluetooth/hci_usb.c
@@ -134,6 +134,8 @@ static struct usb_device_id blacklist_id
 
 	/* Dell laptop with Broadcom chip */
 	{ USB_DEVICE(0x413c, 0x8126), .driver_info = HCI_RESET | HCI_WRONG_SCO_MTU },
+	/* Dell Wireless 370 */
+	{ USB_DEVICE(0x413c, 0x8156), .driver_info = HCI_RESET | HCI_WRONG_SCO_MTU },
 
 	/* Microsoft Wireless Transceiver for Bluetooth 2.0 */
 	{ USB_DEVICE(0x045e, 0x009c), .driver_info = HCI_RESET },
